recipes/conversational_voice_agent/load_tts.py

```diff
@@ -0,0 +1,26 @@
+import nltk
+from nltk.tokenize import word_tokenize
+
+# Attempt to download the 'punkt' tokenizer
+try:
+    nltk.data.find('tokenizers/punkt')
+except LookupError:
+    nltk.download('punkt')

```

Comment:
Put it into separate function and call it from main

Reply:
Moved the NLTK to app.py